src/Analyzer/Pass/Statement/TestAnnotation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PHPSA\Analyzer\Pass\Statement;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPSA\Analyzer\Pass\AnalyzerPassInterface;
+use PHPSA\Analyzer\Pass\ConfigurablePassInterface;
+use PHPSA\Context;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use phpDocumentor\Reflection\DocBlock;
+
+class TestAnnotation implements ConfigurablePassInterface, AnalyzerPassInterface
+{
+    /**
+     * @param ClassMethod $methodStmt
+     * @param Context $context
+     * @return bool
+     */
+    public function pass(ClassMethod $methodStmt, Context $context)
```

Comment:
Please don't name variable for own `Definitions` like `$methodStmt`
It's not a method statement, it's a `$method`